packages/indexer/lib/index.js

```diff
@@ -191,7 +193,7 @@ class TransactionCollector {
   constructor(
     indexer,
     { lock = null, type = null } = {},
-    { skipMissing = false, includeStatus = true } = {}
+    { skipMissing = false, includeStatus = true, fromBlock = null, toBlock = null } = {},
```

Comment:
`fromBlock` and `toBlock` should be in the query object together with lock and type, not options here.

Reply:
ok, I have move the `fromBlock | toBlock` to the same field in `lock | type`.